docs/application/native/guides/multimedia/media-playback.md

```diff
@@ -69,13 +65,13 @@ Tizen enables your application to [play video](#play_video), and it uses its own
 
 The supported video formats include WMV, ASF, MP4, 3GP, AVI, MKV, and OGG. The available formats depend on the target device.
 
-In the case of a video interruption, the state can be READY due to the resource restriction in the system.
+In the case of a video interruption, the state will be one of IDLE, READY or PAUSED due to the resource restriction in the system. You must check the state of the current instance and take appropriate action.
```

Comment:
Suggest rephrasing, please verify:

```suggestion
In case there is a video interruption, the state will be either IDLE, READY, or PAUSED because of the resource restriction in the system. You must check the state of the current instance and take appropriate action.
```